content/app-dev/workflow/_index.en.md

```diff
@@ -17,10 +17,11 @@ Once you've established a dependency on a library that exposes the interface abs
 The developer's _iteration loop_ for building an actor looks something like this:
 1. Make code changes
 1. Compile and sign the WebAssembly module, creating a `_s.wasm` file (using `make` in an actor project)
-1. Deploy the module to public or local OCI registry (`make push`)
-1. Test the module in a host
-    1. Manually via uploading in wasmCloud web dashboard or with (`make start`)
-    1. Script `wash call` commands to invoke the actor
+1. Test the module
+    1. In the wasmCloud web dashboard
+        1. Use the _From File (Hot reload)_ form to automatically watch actor files for changes
+        1. Script `wash call` commands to invoke the actor
```

Comment:
outdent "Script ..." and "Modify your .."

Reply:
I actually think those belong there, we pretty much have 2 ways to test at the moment which is launching the actor in a host and invoking it, or using the `wasmcloud:testing` interface. Those script + modify steps are supposed to be a part of the development feedback loop after you hot reload the actor